plugins/ModuleLevelTrigger.cpp

```diff
@@ -148,10 +149,22 @@ ModuleLevelTrigger::create_decision(const triggeralgs::TriggerCandidate& tc)
   decision.trigger_number = m_last_trigger_number + 1;
   decision.run_number = m_run_number;
   decision.trigger_timestamp = tc.time_candidate;
-  // TODO: work out what to set this to
-  decision.trigger_type = 1; // m_trigger_type;
   decision.readout_type = dfmessages::ReadoutType::kLocalized;
 
+  // TODO: work out what to set this to
+  if (m_hsi_passthrough == true){
+    if (tc.type == triggeralgs::TriggerCandidate::Type::kTiming){
+      decision.trigger_type = tc.detid;
```

Comment:
Kurt suggests masking off the upper bits of `tc.detid`, in case they're set, so they don't leak into the wrong part of the trigger type. Let's do that, so `decision.trigger_type = tc.detid & 0xff;`

Reply:
added